src/Liquidator.sol

```diff
@@ -431,17 +444,22 @@ contract Liquidator is Owned, ILiquidator {
         } else if (block.timestamp > auctionInformation_.cutoffTimeStamp) {
             // Unhappy flow: Auction did not end within the cutoffTime.
             ILendingPool(creditor).settleLiquidationUnhappyFlow(account, startDebt, msg.sender);
-            // All remaining assets are transferred to the owner of Liquidator.sol,
-            // And a manual (trusted) liquidation has to be done.
-            IAccount(account).auctionBoughtIn(owner);
+            // All remaining assets are transferred to the asset recipient,
+            // and a manual (trusted) liquidation has to be done.
+            IAccount(account).auctionBoughtIn(assetRecipient);
         } else {
             // None of the conditions to end the auction are met.
             return false;
         }
 
```

Comment:
shouldn't we call _endAuction here as well?

Reply:
Not here since the auction doesn't end here, but good point, it should also be called in the bid... if endAuction_